### PR TITLE
Fix/#1751 sf mega menu on mobile

### LINKED ISSUE #1751: [BUG] SfMegaMenu - bar shows on desktop when in production mode
**Describe the bug**
When in production mode SfMegamenu shows the bar in the desktop view but it should be visible only in mobile view.

**How to reproduce**
1. run in production mode
2. go to header with SfMegamenu
3. see the bar showing in desktop mode. 

**Expected behavior**
Bar shoudl be hidden in desktop view. 

**Actual behavior**

**Code examples**
Bug appears because SfBar classes has greater specifity than custom properties in SfMegamenu, so this could help:
``` 
.sf-mega-menu__bar.sf-bar {
  display: flex;
  @include for-desktop {
    display:  none;
  }
}
```

**Screenshots**

image

**Devices (please complete the following information):**
 - Device: [e.g. iPhone6, Desktop]
 - Browser: [e.g chrome, safari][version]

**Additional context**
Anything you would like to add but was not sure which section it fits. For example: tech stack used (Nuxt, Vue, Gridsome, React, etc)

### PR DESCRIPTION
**Pull request by Baroshem**
# Related issue

Closes #1751 

# Scope of work

# Screenshots of visual changes

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/organisms/SfMegaMenu/SfMegaMenu.stories.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @import "~@storefront-ui/shared/styles/helpers";
-.sb-mega-menu {
+.sf-mega-menu {
```
Comment (justyna-13) [inline packages/vue/src/components/organisms/SfMegaMenu/SfMegaMenu.stories.scss:2]: this class name has a different prefix and it's not a mistake :slightly_smiling_face:

File: packages/vue/src/components/organisms/SfMegaMenu/SfMegaMenu.stories.scss [hunk 2]
```diff
@@ -20,4 +20,10 @@
       }
     }
   }
+  &__sf-bar {
```
Comment (justyna-13) [inline packages/vue/src/components/organisms/SfMegaMenu/SfMegaMenu.stories.scss:23]: I think this isn't the right file for this change. What do you think about relocating this part to `SfMegaMenu`? :)

**Review by justyna-13 — COMMENTED** [review]
See my comments @Baroshem :)

**Review by justyna-13 — APPROVED** [review]

**Comment by Baroshem** [comment]
Closed due to other, more appropriate PR

**Pull request closed without merging**